app/actions/app_apply_manifest.rb

```diff
@@ -156,7 +156,11 @@ def create_service_bindings(manifest_service_bindings_message, app)
             message: binding_message)
 
           begin
-            result = action.bind(binding)
+            result = if binding_message.parameters
+                       action.bind(binding, parameters: binding_message.parameters)
+                     else
+                       action.bind(binding)
+                     end
```

Comment:
The if statement is needed here. `binding_message.parameter` will be set to an empty object if not set in the manifest and `action.bind` can handle that.

Reply:
Good point!